### ISSUE TITLE
frame rate of angle yaw pitch Roll is 100 max, can it be higher.

### ISSUE THREAD

**Issue by priyanshuJainGit** [jrowberg/i2cdevlib#474]
how can we increase the data rate if its possible

**Comment by ZHomeSlice** [comment]
Yes, It is possible to have the DMP trigger at a rate of 200 samples a second. Note that the DMP samples the Gyros and accelerometers at a higher sample rate and stabilizes them for you if you wish to sample even higher rate than 200 you will need to directly access the data and process it off the MPU

You will need to alter a byte or two in the DMP firmware to dump data into the FIFO Buffer at the higher rate. This is not as easy as it may seem.

The older version has a spot in the code to make these changes. In the Newer DMP 6.12, I didn't add any code to directly alter this memory location you will need to investigate it at invensense:
[https://www.invensense.com/developers/login/](https://www.invensense.com/developers/login/)
Create an account and login then search the page for MPU6050 

I am not sure, but I believe that the 100 samples a second to be more accurate as the filtering has more readings to stabilize the angles. Note that even at lower DMP Rates the DMP directly samples the Gyros and Accelerometer at the same higher rate, it then filters and merges the data to get the same results.

Z
